.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -81,9 +88,16 @@ module.exports = (actionInfo) => {
         await rootSpan
           .traceChild('prepare packages for packing')
           .traceAsyncFn(async () => {
+            const repoData = require(path.join(repoDir, 'package.json'))
+
             for (const pkg of pkgs) {
               const pkgPath = path.join(repoDir, 'packages', pkg)
-              const packedPkgPath = path.join(pkgPath, `${pkg}-packed.tgz`)
+              const packedPkgPath = path.join(
```

Comment:
Can this use a different stable directory, otherwise it'll bloat the core files. E.g. put it in `test/tmp` or something?